server/api/downloads/key/[requestid].get.ts

```diff
@@ -24,15 +24,13 @@ export default defineEventHandler(async (event) => {
     });
   }
 
-  const data = request.download_key as string;
+  const keyData = request.download_key as string;
+  const decodedKey = atob(keyData);
```

Comment:
One thing we may want to check is that the decoded key has a linefeed at the end - I think I added that in the copy implementation because I discovered that SSH keys would be determined to be invalid if they were missing that for whatever reason.
```suggestion
  const decodedKey = atob(keyData) + '\n';
```